packages/okta-vue/test/e2e/harness/test/e2e/nightwatch.conf.js

```diff
@@ -8,11 +8,13 @@ module.exports = {
 
   selenium: {
     start_process: true,
-    server_path: require('selenium-server').path,
+    // server_path: require('selenium-server').path,
```

Comment:
Slate comment here and a few lines below?

Reply:
oh. duh. comment should go.